Habitica/src/main/java/com/magicmicky/habitrpgwrapper/lib/models/Skill.java

```diff
@@ -20,5 +20,8 @@
     @Column
     public Integer mana, lvl;
 
+    @Column
```

Comment:
The @Column annotation is used to define which attributes are persisted in the database. The temporary response values shouldn't be persisted.